### PR TITLE
Contribution

### LINKED ISSUE #125: I want to add a calculator with more than just basic functions using C language
Please include a small description or summary of the issue here.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Additional context**

**Languages**
- [ ] Python
- [ ] Java
- [x] C
- [ ] C++

I will be adding a calculator with sqrt, log, power func along with basic operations using C language
[c calculator for github.txt](https://github.com/kaustubhgupta/Hacktoberfest2020_/files/5319449/c.calculator.for.github.txt)

### PR DESCRIPTION
**Pull request by adilsp5075**
# Title/Heading < Edit this part>

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. < Edit this part>

Fixes #125

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes
-   [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by adilsp5075** [comment]
menu driven calculator with extra two functions

File: README.md [hunk 1]
```diff
@@ -32,3 +32,5 @@ Voila!
 ## Happy?? Do Star ⭐ this Repo. 🤩
```
Comment (kaustubhgupta) [inline README.md:32]: I have not asked to edit this

File: my calc/my calculator.c [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
```
Comment (kaustubhgupta) [inline my calc/my calculator.c:1]: Put the files (image and code) in the appropriate directory (C folder) and edit that folder readme with your entry (you will understand once you look at it)

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by adilsp5075** [comment]
Sorry for the inconvenience I will check it out

**Comment by kaustubhgupta** [comment]
You made another PR, this will be considered invalid

**Pull request closed without merging**